### DISCUSSION TITLE
LMS

### DISCUSSION THREAD

**Discussion by VaBanck** [sle118/squeezelite-esp32#356] [General]
![Снимок экрана 2023-11-08 в 19 34 11](https://github.com/sle118/squeezelite-esp32/assets/75133860/1cf0655d-7a47-47d9-b9a9-fb11df193b0b)
How can I uninstall LMS on my mac, I need to reinstall it, it doesn't work correctly

**Comment by philippe44** [comment]
Please go there https://forums.slimdevices.com/
